nl/devlink_linux.go

```diff
@@ -0,0 +1,36 @@
+package nl
+
+const (
```

Comment:
can you put a reference of the linux kernel from where these values are defined?

Reply:
@fcrisciani sure, reference in code or in commit log? for example nl/tc_linux.go doesn't have any reference for value definition for TCA_POLICE_TBF etc.
But that may be old. I can add the reference to kernel uapi header file. Let me know.

Reply:
look at https://github.com/vishvananda/netlink/blob/master/nl/conntrack_linux.go, I guess there is value in having something like that so that there is a clear reference on where the values are coming from

Reply:
@fcrisciani sure. I will add it.